data/human/culture conversations.txt

```diff
@@ -622,3 +622,72 @@ mission "Sol-Bound Shuttle"
 			label end
 			`	You say your goodbyes as you board your ships. Although you are both captains, the trajectory of your lives have taken very different courses.`
 				decline
+
+# Note about Earth Day and Earth Hour:
+# Earth Day (in 2023) is a day of awareness about climate change and occurs every 22nd of April. Earth Hour is an hour
+# of blackout around cities worldwide, also to bring awareness to climate change, and it's observed on the last Saturday
+# of March.
+
+# Since Earth Day has become a quasi-'national day' for Earthlings in Endless Sky, it would be interesting to have Earth
+# Hour be conflated with Earth Day as well, to demonstrate how people in a thousand years' time observe what we observe
+# today and how its meaning has changed.
+mission "Earth Day Blackout"
+	minor
+	landing
+	source Earth
+	to offer
+		day = 22
+		month = 5
+	on offer
+		conversation
+			`As you follow the tracking beacons towards Earth's central spaceport, you realize there's something very off; what was the bustling spaceport city is now completely still, and even though it is near midnight, the supertall buildings and skyscrapers are curiously dark; not a light is on for kilometers.`
+			`	However, the spaceport seems to still be operating, because the standard, automated procedures of signing landing documents, refuelling and recalibrating, and checking your ship's spaceworthiness were met and completed as you landed. But when you enter the spaceport terminal, there are wax candles spaced out to bring some light into the now dark corridors. It's a strange sight for the most populated planet in human space.`
```

Comment:
The thought was that the player would see the operational spaceport and think, 'The spaceport must still have their lights on if they can continue to service ships,' and then when they enter the terminal the lights aren't on anyway. But I think it will be better to change it.